java/src/org/openqa/selenium/firefox/AddHasExtensions.java

```diff
@@ -28,12 +28,15 @@
 import org.openqa.selenium.remote.ExecuteMethod;
 import org.openqa.selenium.remote.http.HttpMethod;
 
+import java.io.FileOutputStream;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
+import java.nio.file.*;
```

Comment:
IntelliJ tends to do this by default when optimizing imports. But it is good practice to avoid importing entire package.

Reply:
Alright, I'll correct that.